library/edihistory/test_edih_835_accounting.php

```diff
@@ -434,7 +434,7 @@ function edih_835_accounting($segments, $delimiters)
     $loopid = '2110';
```

Comment:
This script makes me a bit nervous since there is no authentication happening here and stuff is happening outside the function. Recommend placing the below at line 25:
```
// comment out below exit when need to use this script
exit;
```